src/languageSupport/languages/flux/lsp/connection.ts

```diff
@@ -137,24 +134,6 @@ class LspConnectionManager {
     })
   }
 
-  _setEditorSyncToggle() {
-    setTimeout(() => {
-      const clickableInvisibleDiv = document.getElementById(ICON_SYNC_ID)
-      // add listeners
-      clickableInvisibleDiv.removeEventListener('click', () =>
-        this._setSessionSync(!this._session.composition.synced)
-      ) // may have existing
-      clickableInvisibleDiv.addEventListener('click', () => {
-        event('Toggled Flux Sync in editor', {
-          active: `${!this._session.composition.synced}`,
-        })
-        this._setSessionSync(!this._session.composition.synced)
-      })
-
-      this._alignInvisibleDivToEditorBlock()
-    }, 1000)
-  }
```

Comment:
From [this](https://github.com/influxdata/ui/issues/5573#issuecomment-1242424106) conversation, we decide to make composition sync icon un-clickable because 1) Monaco editor doesn't provide such API, 2) there is already a toggle nearby that turns on/off composition sync